### PR TITLE
 Handle dataset improvement publishing

### PR DESCRIPTION
**Pull request by maukoquiroga**
partial_fix #158 

It does basically creates a context for data improvement, including improved dataset uploads.

### CODE REVIEW

File: lib/transport/reusable_data/licence.ex [hunk 1]
```diff
@@ -5,41 +5,47 @@ defmodule Transport.ReusableData.Licence do
...
-        %__MODULE__{name: dgettext("reusable_data", "odc-odbl")}
-      "other-open" ->
-        %__MODULE__{name: dgettext("reusable_data", "other-open")}
-      "notspecified" ->
-        %__MODULE__{name: dgettext("reusable_data", "notspecified")}
-      _ ->
-        %__MODULE__{name: nil}
-    end
+    attrs
+      |> super
```
Comment (l-vincent-l) [inline lib/transport/reusable_data/licence.ex]: indentation

File: lib/transport/reusable_data/licence.ex [hunk 2]
```diff
@@ -5,41 +5,47 @@ defmodule Transport.ReusableData.Licence do
...
+      |> Map.get(:name)
+      |> List.wrap
```
Comment (l-vincent-l) [inline lib/transport/reusable_data/licence.ex]: You could just to 
```
attrs
|> Map.get(:name, [])
```

File: lib/transport/reusable_data/licence.ex (hunk 1 above)
Comment (maukoquiroga) [reply lib/transport/reusable_data/licence.ex]: Good catch

File: lib/transport/reusable_data/licence.ex (hunk 2 above)
Comment (maukoquiroga) [reply lib/transport/reusable_data/licence.ex]: What I want to do here is to 1. take the name, 2. create a list, and 3. create a map. In ```Map.get/3```, third argument is just a fallback.

**Pull request merged**